.circleci/config.yml

```diff
@@ -37,6 +37,10 @@ jobs:
           name: use opensource tool client9/misspell to correct commonly misspelled English words
           command: |
             find  ./* -name  "*" | xargs misspell -error
+      - run:
+          name: use yamllint tool to check the format of all yaml files
+          command: |
+            find ./* -name "*" | xargs yamllint -f colored
```

Comment:
I think we just need to check the yaml file, so `find ./* -name "*.yml" | xargs yamllint -f colored` ?